.vscode/settings.json

```diff
@@ -1,9 +1,6 @@
 {
   "editor.formatOnSave": true,
   "solidity.monoRepoSupport": true,
-  "eslint.validate": [
-    "javascript",
-    "svelte",
-    "typescript"
-  ]
-}
\ No newline at end of file
+  "eslint.validate": ["javascript", "svelte", "typescript"],
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
this ensures vscode is using the locally installed TS rather than the one bundled with vscode